howtos/technical-writing-styleguide.md

```diff
@@ -32,23 +35,24 @@ We encourage document authors and technical writers to keep in mind grammar, pun
+| [In to vs. into](https://www.grammarly.com/blog/into-vs-in-to/?&utm_source=google&utm_medium=cpc&utm_campaign=10273012991&utm_targetid=aud-879636892685:dsa-1233402314764&gclid=CjwKCAjw55-HBhAHEiwARMCszrAxBW--e56ITNgN5AVsSot_KSUlLcc6oLBkI0741uFAWldNQeVzrRoCwvIQAvD_BwE&gclsrc=aw.ds)                                                                                                                                                                                                                           | Always think of "into" as a single preposition (within or outside of something.)<br>Typically, "into" refers to physically going inside.<br>Think of "in to" as two independent prepositions that happen to end up next to one another, and typically aren't physically entering or exiting something.<br>Oftentimes, you can tell if you should use "in to" because you could place a comma after "in," and the sentence would still make sense.                                                                                                                                                                                                                                                                                                                                                                   | Type your name in to the text box to sign up for Camunda Platform 8.<br>"Check which folder the file is into ensure you are in the correct location."                                                                                                                   | Type your name into the text box to sign up for Camunda Platform 8.<br>"Check which folder the file is in to ensure you are in the correct location."                                                                             |
```

Comment:
```suggestion
| [In to vs. into](https://www.grammarly.com/blog/into-vs-in-to/)                                                                                                                                                                                                                           | Always think of "into" as a single preposition (within or outside of something.)<br>Typically, "into" refers to physically going inside.<br>Think of "in to" as two independent prepositions that happen to end up next to one another, and typically aren't physically entering or exiting something.<br>Oftentimes, you can tell if you should use "in to" because you could place a comma after "in," and the sentence would still make sense.                                                                                                                                                                                                                                                                                                                                                                   | Type your name in to the text box to sign up for Camunda Platform 8.<br>"Check which folder the file is into ensure you are in the correct location."                                                                                                                   | Type your name into the text box to sign up for Camunda Platform 8.<br>"Check which folder the file is in to ensure you are in the correct location."                                                                             |
```

Unless that is our UTM, I would strip it out.